extern/ProtoGeometry/ProtoGeometry.config

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0"?>
 <ProtoGeometryConfiguration xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <GeometryFactoryFileName>libg_219/LibG.ProtoInterface.dll</GeometryFactoryFileName>
+  <GeometryFactoryFileName>libg_220/LibG.ProtoInterface.dll</GeometryFactoryFileName>
```

Comment:
It's not obvious to me why we're removing libg 219 support in this PR. Is it possible for this to be a separate PR, review by, for example, @Benglin?